app/controllers/users_controller.rb

```diff
@@ -202,7 +202,7 @@ def check_user_signup_allowed
 
   def user_params
     params.require(:user).permit(:name, :email, :image, :password, :password_confirmation,
-      :new_password, :provider, :accepted_terms, :language)
+      :new_password, :provider, :accepted_terms, :language, :time_zone)
```

Comment:
Not needed

Reply:
Need it there to update the time_zone attribute